### PR TITLE
chore: apply radix colors in UI system

### PR DESCRIPTION
**Pull request by tobi-or-not-tobi**
We had a few places where we did not correctly leverage the colors. 

closes [HRZ-2901](https://spryker.atlassian.net/browse/HRZ-2901)

[HRZ-2901]: https://spryker.atlassian.net/browse/HRZ-2901?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/template/presets/src/b2c/experience/header.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ export const HeaderTemplate: StaticComponent = {
           rules: [
             {
               layout: 'flex',
-              background: 'var(--oryx-color-primary-500)',
```
Comment (tolerants) [inline libs/template/presets/src/b2c/experience/header.ts:37]: Hm, is it a good idea to use direct color value?

**Review by tolerants — DISMISSED** [review]

File: libs/template/presets/src/b2c/experience/header.ts (hunk 1 above)
Comment (tobi-or-not-tobi) [reply libs/template/presets/src/b2c/experience/header.ts:37]: It's an exception; when we use the design token, we get a very ugly header with all colors in dark mode.

**Review by Sinucid — DISMISSED** [review]

**Comment by tobi-or-not-tobi** [comment]
we'll retry

**Comment by tobi-or-not-tobi** [comment]
recreated at #338

**Pull request closed without merging**